lib/puppet/type/concat_fragment.rb

```diff
@@ -1,48 +1,62 @@
 Puppet::Type.newtype(:concat_fragment) do
-  @doc = "Create a concat fragment to be used by concat.
-    the `concat_fragment` type creates a file fragment to be collected by concat based on the tag.
-    The example is based on exported resources.
-
-    Example:
-    @@concat_fragment { \"uniqe_name_${::fqdn}\":
-      tag => 'unique_name',
-      order => 10, # Optional. Default to 10
-      content => 'some content' # OR
-      content => template('template.erb') # OR
-      source  => 'puppet:///path/to/file'
-    }
-  "
+  @doc = <<-DOC
+    @summary
+      Manages the fragment.
+
+    @example
+      # The example is based on exported resources.
+
+      concat_fragment { \"uniqe_name_${::fqdn}\":
+        tag => 'unique_name',
+        order => 10, # Optional. Default to 10
+        content => 'some content' # OR
+        # content => template('template.erb')
```

Comment:
is this meant to be a comment ?

Reply:
Yea the original example had both content parameters defined to show the alternative options you could choose. However since that is syntactically incorrect and would have thrown an error I just commented one of them. I can change it if it seems confusing.